questionpy_server/settings.py

```diff
@@ -21,7 +21,12 @@ def __call__(self, settings: BaseSettings) -> Dict[str, Any]:
 
             parser = ConfigParser()
             parser.read(path)
-            return {'webservice': dict(parser['webservice'])}
+            return {
+                'webservice': dict(parser['webservice']),
+                'package_cache': dict(parser['cache-package']),
+                'question_state_cache': dict(parser['cache-question_state']),
+                'collector': dict(parser['collector'])
+            }
```

Comment:
Können wir vielleicht die INI-Namen und die Feld-Namen aneinander angleichen? Dann könnte man das durch
```suggestion
            return {key: section for key, section in parser.items() if key != "DEFAULT"}
```

o.ä. ersetzen